README.md

```diff
@@ -17,20 +17,26 @@ Probably the most well known textbook on machine learning. Recommended for mathe
 
 * Marina also has a [list of books on probability](http://www.stat.washington.edu/courses/stat391/spring15/books.html), for STAT 391 students. 
 
-* UW faculty member [Jake VanderPlas](http://staff.washington.edu/jakevdp/) is working on an O'Reilly book "Python Data Science Handbook". Some previews are [posted](https://www.oreilly.com/people/89c9c-jake-vanderplas).
+* UW faculty member [Jake VanderPlas](http://staff.washington.edu/jakevdp/) is the author of O'Reilly's "Python Data Science Handbook". All notebooks and resources used in the book are [posted](https://github.com/jakevdp/PythonDataScienceHandbook/tree/master/notebooks).
 
 ## Software
 
-**Either Python or R are both excellent languages to know for those getting started in machine learning. Lectures will initially be taught in R.**
```

Comment:
I still think its worth mentioning that R is an ok language for machine learning, but Python seems to be more on the rise?